apps/plugin-browser/src/pages/popup/popup-contents/action-center/text-field-with-dark-mode.tsx

```diff
@@ -0,0 +1,27 @@
+import { TextField, TextFieldProps } from "@hashintel/design-system";
+import { outlinedInputClasses } from "@mui/material";
+
+export const TextFieldWithDarkMode = (props: TextFieldProps) => {
+  return (
+    <TextField
+      {...props}
+      InputProps={{
+        sx: ({ palette }) => ({
+          "@media (prefers-color-scheme: dark)": {
+            background: "#161616",
+
+            [`.${outlinedInputClasses.notchedOutline}`]: {
+              border: `1px solid ${palette.gray[90]}`,
+            },
+
+            // @todo figure out where these styles should be (not taking effect here)
+            "::placeholder": {
+              color: palette.gray[70],
+            },
+            color: palette.common.white,
```

Comment:
I'm not sure why the `!important` was needed, but this worked for me:
```suggestion
            // @todo figure out where these styles should be (not taking effect here)
           [`.${outlinedInputClasses.input}`]: {
              color: palette.common.white,
              "&::placeholder": {
                color: `${palette.gray[70]} !important`,
              },
            },
```

Reply:
Thank you, done (in follow-up branch) https://github.com/hashintel/hash/commit/e8f7fc9bd0044ef9b260b26c5fcc8b281aa2bbd7